code/modules/mob/living/simple_animal/abnormality/teth/my_sweet_home.dm

```diff
@@ -0,0 +1,96 @@
+//Brain go brrrr.
+/mob/living/simple_animal/hostile/abnormality/my_sweet_home
+	name = "My Sweet Home"
+	desc = "This cozy little house is a safe nest built only for you. Everything is here for you..."
+	icon = '---'
```

Comment:
Don't set icon to file that doesn't exist. You can remove it entirely or set it to null instead.

Reply:
What's null for this coding language?

Reply:
null is well, null? Nothingness.
if(null) will return FALSE, for example.